rstudio/userconf.sh

```diff
@@ -5,29 +5,27 @@ USER=${USER:=rstudio}
 PASSWORD=${PASSWORD:=rstudio}
 USERID=${USERID:=1000}
 ROOT=${ROOT:=FALSE}
-DEL_DEFAULT_USER=${DEL_DEFAULT_USER:=FALSE}
 
-if [ "$USERID" -ne 1000 ]
+if [ $USERID -ne 1000 ]
```

Comment:
Maybe keep the quotes if you keep them in all the other variable expansions below?  Just wondering...